Source/PP_Spray/SprayInjection.H

```diff
@@ -204,9 +226,17 @@ SprayParticleContainer::uniformSprayInit(
   }
   part_vals[SprayComps::pstateT] = T_part;
   part_vals[SprayComps::pstateDia] = dia_part;
-  for (int spf = 0; spf < SPRAY_FUEL_NUM; ++spf) {
```

Comment:
Was this removed intentionally? Test cases in LMeX and C are failing with NaNs due to these components not being initialized.

Reply:
I believe this was mistakenly removed and should be added back in. Good catch

Reply:
Thanks for the quick replies. This has been done in https://github.com/AMReX-Combustion/PelePhysics/pull/452